### DISCUSSION TITLE
NocoDB on a 8 Million row postgres db [May 08]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2055] [Discord Threads]
> 🙋‍♂️ **CraigGrella**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### I wouldn't see much impact on performance.   
####    
#### The data retrieved is paginated  
#  
> 💁 **o1lab**  
#### if some views are slow based on filters applied (since rows is in million) - you would have to create index on such tables  
#  
> 🙋‍♂️ **CraigGrella**  
#### They fields I need indexed are done in db already. Ty for the quick reply.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/969944531808170014)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
